app/models/case/sar/offender_complaint.rb

```diff
@@ -8,17 +8,28 @@ class Case::SAR::OffenderComplaint < Case::SAR::Offender
   jsonb_accessor :properties,
                  complaint_type: :string,
                  complaint_subtype: :string,
+                 ico_contact_name: :string,
+                 ico_contact_email: :string,
+                 ico_contact_phone: :string,
+                 ico_reference: :string,
+                 gld_contact_name: :string,
+                 gld_contact_email: :string,
+                 gld_contact_phone: :string,
+                 gld_reference: :string,
                  priority: :string
 
   validates :complaint_type, presence: true
   validates :complaint_subtype, presence: true
   validates :priority, presence: true
+  validate :validate_ico_contact_name
+  validate :validate_ico_contact_details
+  validate :validate_ico_reference
   validate :validate_external_deadline
 
   enum complaint_type: {
-    standard_complaint: 'Standard',
-    ico_complaint: 'ICO',
-    litigation_complaint: 'Litigation',
+    standard_complaint: 'standard_complaint',
+    ico_complaint: 'ico_complaint',
+    litigation_complaint: 'litigation',
   }
 
```

Comment:
sounds like you still choose to define the wording in yaml file :),   in that case,  I need to change the coding in the filters otherwise the wording for filters won't be correct.

Reply:
Yeah, I think there's value in following the same pattern unless there's an overwhelming reason to depart from it, which is why I've changed this back to work like other enums in the app - felt like a lot of fiddling, but I think this is the right way to go :)

Reply:
@Polsonby  ok for me 👍 🤓